README.md

```diff
@@ -18,7 +18,8 @@ All talks presented at PyData Delhi 2017:
 |11|Difficulties in working with the government data and how can tech help us resolve it| Ankita Mathur| - | - |_in process_|
 |12|Using RNNs to implement a Full Adder using PyTorch| Madhur Tandon| - | - |_in process_|
 |13|Deriving insights in minutes from serverless IoT Data Pipeline| Narendran R| - | - |_in process_|
-|14|Machine Learning Architectures | Shagun Sodhani| - | @shagunsodhani  |_in process_|
+|14|Machine Learning Architectures | Shagun Sodhani| [Machine Learning Architectures](http://slides.com/shagunsodhani/pydata2017#/) | [@shagunsodhani](https://twitter.com/shagunsodhani)
+  |_in process_|
```

Comment:
not next line :/

Reply:
@Dawny33 😠

Reply:
@manojpandey https://github.com/pydatadelhi/delhi2017/pull/6